finalProject/tenSpeckledFrogs.js

```diff
@@ -0,0 +1,25 @@
+//Part 1:
+
+for (var i = 10; i > 0; i--) {
```

Comment:
i++ is more commonly used, though I see why it makes sense to use -- given the extra math that would be needed to get the number to print.